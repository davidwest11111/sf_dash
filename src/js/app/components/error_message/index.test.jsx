import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import ErrorMessage from './index';


describe('outputs the expected tree when', () => {
  let wrapper;

  test('no children', () => {
    wrapper = shallow((
      <ErrorMessage />
    ));
  });

  test('children', () => {
    wrapper = shallow((
      <ErrorMessage>
        some error message
      </ErrorMessage>
    ));
  });

  afterEach(() => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
