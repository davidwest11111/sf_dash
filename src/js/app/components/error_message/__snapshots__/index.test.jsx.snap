// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outputs the expected tree when children 1`] = `
<div
  className="error-message"
>
  <span
    className="fas fa-exclamation-circle mr-1"
  />
  <span>
    some error message
  </span>
</div>
`;

exports[`outputs the expected tree when no children 1`] = `""`;
